src/main/java/com/google/ehub/data/EntertainmentItemDatastore.java

```diff
@@ -0,0 +1,104 @@
+package com.google.ehub.data;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.CompositeFilterOperator;
+import com.google.appengine.api.datastore.Query.FilterOperator;
+import com.google.appengine.api.datastore.Query.FilterPredicate;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.ehub.utility.NextIncreasingLexicographicStringUtility;
+import java.util.ArrayList;
+import java.util.List;
+
+
+/**
+ * Manages EntertainmentItems stored in Datastore.
+ */
+public final class EntertainmentItemDatastore {
+  public static final String ENTERTAINMENT_ITEM_KIND = "entertainmentItem";
+  public static final String TITLE_PROPERTY_KEY = "title";
+  public static final String LOWERCASE_TITLE_PROPERTY_KEY = "lowercaseTitle";
+  public static final String DESCRIPTION_PROPERTY_KEY = "description";
+  public static final String IMAGE_URL_PROPERTY_KEY = "imageURL";
+
+  private static DatastoreService datastoreService = DatastoreServiceFactory.getDatastoreService();
+
+  /**
+   * Adds an EntertainmentItem to Datastore.
+   *
+   * @param title title of the entertainment item
+   * @param description description of the entertainment item
+   * @param imageURL URL to the image of the entertainment item stored in Blobstore
+   */
+  public static void addEntertainmentItemToDatastore(
+      String title, String description, String imageURL) {
```

Comment:
API-wise, I think we should maintain consistency of what items are going in and out of this class. 

My recommendation is using EntertainmentItem everywhere instead of having callers pass in individual fields.

Reply:
I changed method so it now only accepts EntertainmentItem as a parameter